docs/requirements.md

```diff
@@ -2,88 +2,78 @@
+| Kubernetes Small Installation            | 16            | 32 GB          | 1                      | 2                                  | 4              | 8 GB            | 10                     | 20 GB                   |
+| Kubernetes Medium Installation (default) | 32            | 64 GB          | 2                      | 4                                  | 8              | 16 GB           | 20                     | 40 GB                   |
+| Kubernetes Big Installation              | 60+           | 110+ GB        | 2+                     | 10+                                | 8+             | 16+ GB          | 50+                    | 100+ GB                 |
 
-Given the previous values, the total number of resources required should be the "per-analysis" amount times the number of concurrent analysis.
+**NOTE:**
+For microk8s we added 1.5 CPU and 1.5 GB extra in the "Platform" meant to be used by microk8s itself.
 
-### Example
+| Installation type              | ~ Total vCPUs | ~ Total Memory | Replicas per component | Max. commits analyzed concurrently | Platform vCPUs | Platform Memory | Analysis Workers vCPUs | Analysis Workers Memory |
+| ------------------------------ | ------------- | -------------- | ---------------------- | ---------------------------------- | -------------- | --------------- | ---------------------- | ----------------------- |
+| MicroK8s Minimum               | 16            | 32 GB          | 1                      | 2                                  | 5.5            | 9.5 GB          | 10                     | 20 GB                   |
+| MicroK8s Recommended (default) | 20+           | 32+ GB         | 1-2                    | 2                                  | 11+            | 20+ GB          | 10                     | 20 GB                   |
 
-_Maximum of 6 concurrent analysis_
+### Storage
 
-```yaml
-worker-manager:
-  config:
-    workers:
-      dedicatedMax: 3
-    workerResources:
-      limits:
-        cpu: 1
-        memory: "2Gi"
-    pluginResources:
-      requests:
-        cpu: 0.5
-        memory: 1000000000 # 1GB
-```
+The storage requirements depend mainly on the number of repositories you will be analyzing.
+Use the following table as a guideline to determine Codacy's storage requirements.
 
-In this example the minimum number of resources required would be:
+| Component | Bundled in the chart         | Minimum Recommended |
+| --------- | ---------------------------- | ------------------- |
+| NFS       | Yes                          | 200 GB              |
+| RabbitMQ  | Yes                          | 8 GB                |
+| Minio     | Yes                          | 20 GB               |
+| Postgres  | No (external DB recommended) | 500 GB+             |
```

Comment:
It seems maybe a bit exaggerated for a MicroK8s installation. Should we split?

Reply:
I used the doplicated as a reference for the postgres values